core/math/math_funcs.h

```diff
@@ -270,10 +270,10 @@ class Math {
 	static _ALWAYS_INLINE_ float move_toward(float p_from, float p_to, float p_delta) { return abs(p_to - p_from) <= p_delta ? p_to : p_from + SIGN(p_to - p_from) * p_delta; }
 
 	static _ALWAYS_INLINE_ double linear2db(double p_linear) { return Math::log(p_linear) * 8.6858896380650365530225783783321; }
-	static _ALWAYS_INLINE_ float linear2db(float p_linear) { return Math::log(p_linear) * 8.6858896380650365530225783783321; }
+	static _ALWAYS_INLINE_ float linear2db(float p_linear) { return Math::log(p_linear) * (float)8.6858896380650365530225783783321; }
```

Comment:
Cant we trust that all compilers will optimize this? Otherwise we could just use the float conversion of this number from the start.

Reply:
Casting should be fine afaik. I did start by using the `.f` form, but didn't know what the closest float equivalent was, and this seemed like an easy way to dodgy that problem. :grin: